hadoop-crypto/src/test/java/com/palantir/hadoop/ConfigurableEncryptedFileSystemTest.java

```diff
@@ -167,6 +169,20 @@ public void testFileNameCollidesWithKeyMaterial() throws IOException {
         efs.open(path);
     }
 
+    @Test
+    public void testListStatus_keyMaterialFilesFiltered() throws IOException {
```

Comment:
a FileSystem higher up intercepted the listStatus calls and never directed them at the backing store